Clouds/shaders/fragment.glsl

```diff
@@ -149,6 +149,13 @@ float raymarchCloud(vec3 cameraPos, vec3 rayDir, float dstInBox, float dstToBox)
     // TODO: Blue noise offset of samplePoint
     vec3 samplePoint = cameraPos + dstToBox * rayDir;
 
+    // texture is smaller that the render target, multiply by 5 to wrap the tex
+    vec3 boxPoint = cloudsBoxCenter - samplePoint;
+    vec2 texCoords = boxPoint.xz / cloudsBoxSideLength + 0.5f;
+
+    vec4 blueNoiseValue = texture(blueNoiseTexture, texCoords);
```

Comment:
to na pewno poprawne koordynaty do samplowania `blueNoiseTexture`? 🤔
![image](https://github.com/daveedoo/volumetric-clouds/assets/73546387/6ffcffbd-6f53-4665-aeeb-9ac4beb07dbc)